handler/handler.go

```diff
@@ -4,11 +4,17 @@ import (
 	"encoding/json"
 	"fmt"
 	"log"
+	"time"
 
 	"github.com/nats-io/go-nats-streaming"
 	"github.com/openfaas/faas/gateway/queue"
 )
 
+var (
+	maxReconnect = 5
```

Comment:
Those two should probably should be configurable.

Reply:
Added that to NatsConfig